exec/jdbc/pom.xml

```diff
@@ -100,6 +100,11 @@
       <groupId>xalan</groupId>
       <artifactId>xalan</artifactId>
     </dependency>
+    <dependency>
```

Comment:
aaahh, I thought it was included, but I searched for `@NotNull` instead and didn't find it! So, yes, should totally be included as a dependency!